pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/MergeRollupMinionClusterIntegrationTest.java

```diff
@@ -97,7 +100,8 @@ public void setUp()
     TableConfig singleLevelConcatTableConfig =
         createOfflineTableConfig(SINGLE_LEVEL_CONCAT_TEST_TABLE, getSingleLevelConcatTaskConfig());
     TableConfig singleLevelRollupTableConfig =
-        createOfflineTableConfig(SINGLE_LEVEL_ROLLUP_TEST_TABLE, getSingleLevelRollupTaskConfig());
+        createOfflineTableConfig(SINGLE_LEVEL_ROLLUP_TEST_TABLE, getSingleLevelRollupTaskConfig(),
+            getMultiColumnSegmentPartitionConfig());
```

Comment:
This `MergeRollupMinionClusterIntegrationTest` was started failing once I had added the multiple column segment partitioning config in table config. then I have fixed the `MergeRollupTaskGenerator` to make it run successfully.